extensions/wikia/UserRenameTool/RenameUserLogFormatter.class.php

```diff
@@ -1,104 +0,0 @@
-<?php
```

Comment:
Refactored into UserRenameToolHelper.class.php and UserRenameTool\Process\Processbase